docs/feature_oled_driver.md

```diff
@@ -136,21 +136,23 @@ void oled_task_user(void) {
-|`OLED_SCROLL_TIMEOUT_RIGHT`|*Not defined*    |Scroll timeout direction is right when defined, left when undefined.                                                      |
-|`OLED_IC`                  |`OLED_IC_SSD1306`|Set to `OLED_IC_SH1106` if you're using the SH1106 OLED controller.                                                       |
-|`OLED_COLUMN_OFFSET`       |`0`              |(SH1106 only.) Shift output to the right this many pixels.<br />Useful for 128x64 displays centered on a 132x64 SH1106 IC.|
-|`OLED_BRIGHTNESS`          |`255`            |The default brightness level of the OLED, from 0 to 255.                                                                  |
-|`OLED_UPDATE_INTERVAL`     |`0`              |Set the time interval for updating the OLED display in ms. This will improve the matrix scan rate.                        |
+|Define                             |Default          |Description                                                                                                               |
+|-----------------------------------|-----------------|--------------------------------------------------------------------------------------------------------------------------|
+|`OLED_DISPLAY_ADDRESS`             |`0x3C`           |The i2c address of the OLED Display                                                                                       |
+|`OLED_FONT_H`                      |`"glcdfont.c"`   |The font code file to use for custom fonts                                                                                |
+|`OLED_FONT_START`                  |`0`              |The starting character index for custom fonts                                                                             |
+|`OLED_FONT_END`                    |`223`            |The ending character index for custom fonts                                                                               |
+|`OLED_FONT_WIDTH`                  |`6`              |The font width                                                                                                            |
+|`OLED_FONT_HEIGHT`                 |`8`              |The font height (untested)                                                                                                |
+|`OLED_TIMEOUT`                     |`60000`          |Turns off the OLED screen after 60000ms of keyboard inactivity. Helps reduce OLED Burn-in. Set to 0 to disable.           |
+|`OLED_DRIVER_DISPLAY_FADE`         |*Not defined*    |Enables fade out animation. Use together with `OLED_TIMEOUT`.                                                             |
+|`OLED_DRIVER_DISPLAY_FADE_INTERVAL`|`0`              |The speed of fade out animation, from 0 to 15. Larger values are slower.                                                  |
```

Comment:
```suggestion
|`OLED_FADE_OUT`         |*Not defined*    |Enables fade out animation. Use together with `OLED_TIMEOUT`.                                                             |
|`OLED_FADE_OUT_INTERVAL`|`0`              |The speed of fade out animation, from 0 to 15. Larger values are slower.                                                  |
```

Reply:
@drashna, as you proposed these names, could you comment on this suggestion?

The new names would be more consistent with the existing options for the OLED driver. Here is an excerpt from the docs (using the exisiting names):
|Define                             |Default          |Description                                                                                                               |
|-----------------------------------|-----------------|--------------------------------------------------------------------------------------------------------------------------|                                                           
|`OLED_FONT_WIDTH`                  |`6`              |The font width                                                                                                            |
|`OLED_FONT_HEIGHT`                 |`8`              |The font height (untested)                                                                                                |
|`OLED_TIMEOUT`                     |`60000`          |Turns off the OLED screen after 60000ms of keyboard inactivity. Helps reduce OLED Burn-in. Set to 0 to disable.           |
|`OLED_DRIVER_DISPLAY_FADE`         |*Not defined*    |Enables fade out animation. Use together with `OLED_TIMEOUT`.                                                             |
|`OLED_DRIVER_DISPLAY_FADE_INTERVAL`|`0`              |The speed of fade out animation, from 0 to 15. Larger values are slower.

Reply:
Ya, drop the  _DRIVER portion of those names as Faux mentioned. When I wrote all the original defines, I dropped the _DRIVER as it was verbose, annoying to continually type, and didn't add a whole lot of clarity to the define.